bloom-housing/bloom
# fix: remove opt out preferences from export and lottery

This PR addresses #4282 


## Description

When an application has selected the opt out option in a preference the data is saved as the preference is "claimed" with an option that is equal to the opt out text. Because of this these applications are getting bucketed into those preferences. And also when exporting the column shows the opt out text rather than being empty

## How Can This Be Tested/Reviewed?

This can be tested using the `Elm Village` listing on a fresh seeding of the staging seeds.

1. Seed with staging data `yarn db:setup:staging`
2. Start up the app and open the `Elm Village` listing in the partner portal
3. Either add a new application or update an existing one
  - The `Work in the city` preference has an opt out option so make sure that is selected
4. Close the listing and run the lottery
5. Export the lottery results. 
  - Your new or updated application should be in the first tab of the export, but not in the `Work in the city` tab.
  - The same application should show nothing in the column for that preference
6. Go to the applications tab in the partner portal and export the applications
  - The export shouldn't show anything in the `Work in the city` column for that application

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
